mitxgraders/helpers/mitmath/__init__.py

```diff
@@ -0,0 +1,11 @@
+from mitxgraders.helpers.mitmath.calc import evaluator
```

Comment:
Can you add a docstring pointing out that these are here to facilitate imports for authors?